cmd/pulumi-test-language/tests.go

```diff
@@ -1750,6 +1750,25 @@ var languageTests = map[string]languageTest{
 			},
 		},
 	},
+	"l2-explicit-parameterized-provider": {
+		providers: []plugin.Provider{&providers.ParameterizedProvider{}},
+		runs: []testRun{
+			{
+				assert: func(l *L,
+					projectDirectory string, err error,
+					snap *deploy.Snapshot, changes display.ResourceChanges,
+				) {
+					requireStackResource(l, err, changes)
+					stack := snap.Resources[0]
```

Comment:
Worth asserting that the explicit provider is registered as a resource?

Reply:
Yeh will do